.gitignore

```diff
@@ -1,3 +1,5 @@
+jupyterlab/labbox_ephys_widgets_jp/src/extensions
+jupyterlab/codesync.txt
```

Comment:
We also ignore this file which is used to indicate whether we are in a development workspace